examples/gatsby-wordpress-gutenberg-default-example/frontend/gatsby-config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  siteMetadata: {
+    title: `Gatsby Default Starter`,
+    description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
+    author: `@gatsbyjs`,
+  },
+  plugins: [
+    `gatsby-transformer-sharp`,
+    `gatsby-plugin-sharp`,
+    {
+      resolve: `gatsby-source-wordpress-gutenberg`,
+      options: {
+        linkOptions: {
```

Comment:
should we rename this to just: 

```
options: {
  url: '...',
}
```

I think thats cleaner than linkOptions -> uri. If we need "option categories" because it gets too much, we can still add it later I guess.

Reply:
Ah I see. You refer to the http-link options. Then maybe it would make sense to put some comments in here to refer to the docs of apollos http link: https://www.apollographql.com/docs/link/links/http/#options
Or we write out the option descriptions ourselves.

Reply:
Yes, that's because you can also add headers or all the the other options of the apollo link, not just the uri.

Reply:
```suggestion
        linkOptions: {
```

- [ ] add comment that this indicates all options passed to apollo-link

Reply:
Well I will probably rewrite this later, so I'll leave it as it is for now.